kubernetes/kube.libsonnet

```diff
@@ -94,13 +94,13 @@
     std.parseInt(std.substr(n, 0, n_len - convert[0])) * convert[1]
   ),
 
-  _Object(apiVersion, kind, name, app=null, namespace=null):: {
+  _Object(apiVersion, kind, name, app=null, namespace=null, short_name=null):: {
     apiVersion: apiVersion,
     kind: kind,
     metadata: {
       annotations: {},
       labels: {
-        name: name,
+        name: if short_name != null then short_name else name,
```

Comment:
You could just do:

```
_Object(x, y, z) {
  metadata+: {
    labels+: {
      name: '',
    },
  },
}
```

Reply:
That removes the need for a new function arg

Reply:
Closing, thank you. You have just increased my jsonnet kunfu substantially.